### DISCUSSION TITLE
Create Game - Create a Series of games

### DISCUSSION THREAD

**Discussion by BoldBigflank** [BattlesnakeOfficial/feedback#216] [Feature Requests]
I would like to run games repeatedly to get some statistics on my snakes. I think what would help is being able to set up a Series on the site.

The simplest way to do this would be to set up a game as expected, but have an option to "Run this game ______ times", to set up a series of games.

Something more advanced would look like a mini version of the tournament page, allowing users to set different end conditions, like "first to 4 wins".

It would make sense to lock in only your own snakes in this type of rapid-fire thing. Perhaps it could include a shareable link to let others agree to join it. It would be a nice way to promote collaboration.

**Comment by chris-bsnake** [comment]
As you mentioned, I think this works as part of the current Battlesnake Tournament system and we could look at making it a bit easier to spin up a basic "Finals, first to N Wins" tournament and then "auto-run" it to get the same effect.
